vtr_flow/tasks/regression_tests/vtr_reg_weekly/koios_no_complex_dsp/config/config.txt

```diff
@@ -0,0 +1,43 @@
+#
+############################################
+# Configuration file for running experiments
+##############################################
+
+# Path to directory of circuits to use
+circuits_dir=benchmarks/verilog/koios
+
+# Path to directory of architectures to use
+archs_dir=arch/COFFE_22nm
+
+# Directory containing the verilog includes file(s)
+includes_dir=benchmarks/verilog/koios
+
+# Add circuits to list to sweep
+circuit_list_add=clstm_like.small.v
+circuit_list_add=clstm_like.medium.v
+circuit_list_add=clstm_like.large.v
+circuit_list_add=lstm.v
+circuit_list_add=gemm_layer.v
+circuit_list_add=tpu_like.medium.v
+circuit_list_add=tiny_darknet_like.small.v
+circuit_list_add=tiny_darknet_like.medium.v
+circuit_list_add=dla_like.medium.v
+
+# Add architectures to list to sweep
+arch_list_add=k6FracN10LB_mem20K_complexDSP_customSB_22nm.xml
+
+# Add include files to the list
+include_list_add=complex_dsp_include.v
```

Comment:
Again, doesn't this turn on the advanced dsp block?  Seems to conflict with the name of the task.

Reply:
Right. Fixing this.